INSTRUMENTS/PHIDGET/PHIDGET22/PHIDGET22.py

```diff
@@ -9,14 +9,31 @@ def onVoltageRatioChange(self, voltageRatio):
     print("VoltageRatio [" + str(self.getChannel()) + "]: " + str(voltageRatio))
 
 
-@flojoy
-def PHIDGET22(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
-    """Pressure Measurement with Phidget 22 sensors"""
+@flojoy(deps={"Phidget22": "1.14.20230331"})
+def PHIDGET22(
+    default: OrderedPair = None,
```

Comment:
If this input is optional then the convenient way to define this input parameter is as follows: `default: Optional[OrderedPair] = None`
where `Optional` is imported from 'typing' package